docs/operate/reference/advanced-modules/_index.md

```diff
@@ -46,6 +46,74 @@ Running {{< glossary_tooltip term_id="modular-resource" text="modular resources"
 
 However, if you are unable to use modular resources because you need to host `viam-server` on a non-Linux system or have an issue with compilation, you may need to [implement a custom component and register it on a server configured as a remote](/operate/reference/advanced-modules/custom-components-remotes/) on your machine.
 
+## Package and deploy using Docker
+
+In rare cases, you may need to package and deploy a module using Docker.
+Use cases for this include:
+
+- Your module has complex system dependencies that cannot be installed on a machine.
```

Comment:
*that cannot _easily_ be installed on a machine.

If they can be installed in docker, they can be installed on the machine. It just might be a giant pain.

Reply:
```suggestion
- Your module has complex system dependencies that cannot be easily installed on a machine.
```